realis.go

```diff
@@ -787,25 +827,59 @@ func (r *realisClient) RestartJob(key *aurora.JobKey) (*aurora.Response, error)
 }
 
 // StartJobUpdate updates all instances under a job configuration.
-func (r *realisClient) StartJobUpdate(updateJob *UpdateJob, message string) (*aurora.Response, error) {
+func (r *realisClient) StartJobUpdate(updateJob *UpdateJob, message string) (*aurora.StartJobUpdateResult_, error) {
```

Comment:
Can I ask why `*aurora.Response` got changed to`*aurora.StartJobUpdateResult_`?  And also for `CreateJob` and `CreateService`,  `*aurora.Response` got removed? As a client, we are using `Detail` field from `aurora.Response`

Reply:
I changed them to be more focused due  to the fact that the verification doesn't guarantee that the response will be exactly the same as the one that comes from Aurora.  As such I don't feel comfortable returning the flat out response.

What are we using `Detail` for?

Reply:
By the way, the detail field is also returned as part of the error when relevant: https://github.com/paypal/gorealis/blob/a9d99067ee689d9833fd62e92f8a66aaa64c2da6/retry.go#L231

Is there any time where the Detail is included when a call is successful?

Reply:
> By the way, the detail field is also returned as part of the error when relevant:
> 
> https://github.com/paypal/gorealis/blob/a9d99067ee689d9833fd62e92f8a66aaa64c2da6/retry.go#L231
> 
> Is there any time where the Detail is included when a call is successful?

I guess as long as we are able to parse the error message to `Detail` object, it shouldn't be a big deal then.

Reply:
After thinking about it, I've made the API backwards compatible again. Less churn for everyone involved.